### PR TITLE
Improve text for slider in dialog and help text.

### PR DESCRIPTION
**Pull request by colinl**
Fix misleading text against message passthrough checkbox.
Remove erroneous paragraph from help text.

### CODE REVIEW

File: nodes/ui_slider.html [hunk 1]
```diff
@@ -112,6 +112,4 @@
     <p>The label can also be set by a message property by setting
     the field to the name of the property, for example <code>{{msg.topic}}</code>.</p>
     <p>Setting <code>msg.enabled</code> to <code>false</code> will disable the slider output.</p>
-    <p>Note: An input msg to the slider node will not change the status information displayed unless the slider
```
Comment (dceejay) [inline nodes/ui_slider.html:115]: Why is this here ? the slider will update if you inject a valid number with or without a following node.
please remove

Comment (dceejay) [reply nodes/ui_slider.html:115]: The lack of status when no nodes are connected is because no message is sent. But yes that can be fixed... fist time it has been reported.

Comment (colinl) [reply nodes/ui_slider.html:115]: I am confused, the PR removes the section about the status because for me the status display is not affected by whether the output is connected.

**Comment by dceejay** [comment]
1000 apologies. I must be going red/green colour/blind. Had that totally about face.
All good, just need you to accept the CLA.

**Comment by colinl** [comment]
No problems.
I signed the CLA last night.  I was surprised to be asked to sign the CLA as I was sure I had done it previously. Also it didn't seem to like my email address so I re-verified that.  Should the check for CLA signing in this PR go away by itself?

**Comment by dceejay** [comment]
the PR has to be from the same email as was used to sign up for github and accept the LA... - I think it should auto recheck - but you may need to make a tiny change and repush it.

**Comment by knolleary** [comment]
The email attached to the *commit* needs to one GitHub knows about. So either add that email to your account on GitHub, or redo the commit in a clean branch with your git profile setup to use the right email.

**Comment by colinl** [comment]
It is still saying I haven't signed, but when I click on Details it says I have.

**Comment by colinl** [comment]
The email I used was already attached to my github account but wasn't the primary one and hadn't (apparently) been verified but it seems to have confused github so I will recreate the PR on a new branch using my primary email address.  Very odd as I haven't had this problem before.

**Comment by colinl** [comment]
Superseded by PR #628
